assets/collapsible-content.css

```diff
@@ -0,0 +1,117 @@
+.collapsible-section-layout {
+  padding-bottom: 5rem;
+  padding-top: 5rem;
+}
+
+@media screen and (min-width: 750px) {
+  .collapsible-section-layout {
+    padding-bottom: 7rem;
+    padding-top: 7rem;
+  }
+}
+
+.collapsible-content__media--small {
+  height: 19.4rem;
+}
+
+.collapsible-content__media--large {
+  height: 43.5rem;
+}
+
+@media screen and (min-width: 750px) {
+  .collapsible-content__media--small {
+    height: 31.4rem;
+  }
+
+  .collapsible-content__media--large {
+    height: 69.5rem;
+  }
+}
+
+.collapsible-content__grid {
+  margin-left: 0;
```

Comment:
This is overriding the negative margin set by `.grid` and can cause a slight mis-alignment on the left. Is that desired?

![Kapture 2021-12-23 at 10 06 26](https://user-images.githubusercontent.com/3710548/147258521-e630567d-6d6e-4bee-b112-db43c479d26c.gif)

Reply:
Hmm yeah not sure if this was needed for a specific case 🤔  I will remove it for now as I don't when it's needed.